projects/fsastorefrontlib/src/core/my-account/store/reducers/claim.reducer.ts

```diff
@@ -24,7 +24,7 @@ export function reducer(
     }
 
     case fromAction.LOAD_CLAIM_BY_ID_SUCCESS: {
-      const content = { ...action.payload };
+      const content = action.payload ? { ...action.payload } : null;
```

Comment:
Let's move these reducer modifications to the effect level. 

Here keep:  content = {...action.payload}
In claim.effect for corresponding action return:
    return new fromActions.LoadClaimByIdSuccess(claims ? claims : null);
    
If this is working do it for all cases :)

Reply:
This example don't work. Even though I pass null to LoadClaimByIdSuccess, when it comes to the reducer `{ ...action.payload }` this part ends as empty object which causing the problem.

Reply:
There is one more possible solution with if. So don't throw success action in effect if received object is null. 
In that way we are avoiding empty/null object in our store to be emitted.